plugins/circular-view/src/StructuralVariantChordTrack/models/StructuralVariantChordTrack.js

```diff
@@ -112,6 +100,29 @@ export default pluginManager => {
           self.renderSuccess,
           self.renderError,
         )
+        makeAbortableReaction(
+          self,
+          () => ({
+            assemblyNames: getTrackAssemblyNames(self),
+            adapter: getConf(self, 'adapter'),
+            assemblyManager: getSession(self).assemblyManager,
+          }),
+          async ({ assemblyNames, adapter, assemblyManager }, signal) => {
+            const m = assemblyManager.getRefNameMapForAdapter(
+              adapter,
+              assemblyNames[0],
+              { signal, sessionId: getRpcSessionId(self) },
+            )
+            m.then(r => self.setRefNameMap(r))
+          },
+          {
+            name: `${self.type} ${self.id} getting refNames`,
+            fireImmediately: true,
+          },
+          () => {},
+          () => {},
```

Comment:
can the three callbacks, loading, set loaded, and seterror be implemented here?

Reply:
if it is feasible. especially error catching is a useful thing to make sure to obtain